ERRORCODES.md

```diff
@@ -0,0 +1,73 @@
+# Interaction API error codes
+
+## Client Errors
+| Error Code | Description |
+|--|--|
+| *10001* | The Dev name of the Conversation Channel Definition in the request isn't valid for the provided value in the *"to":"\<channelAddressIdentifier\>"* field. |
+| *10002* | The channel type for the provided *"to":"\<channelAddressIdentifier\>"* field isn't valid. |
+| *10003* | The supplied Authorization access token is missing the required Connected App scope. Required value: interaction_api. |
+| *10004* | The supplied Authorization access token is not valid. |
+| *10005* | The AuthorizationContext for the request isn't valid based on the supplied Authorization access token. |
+| *10006* | The channel provider for the provided *"to":"\<channelAddressIdentifier\>"* field isn't valid |
+| *10007* | The file ID field is empty|
+| *10008* | The file size exceeds the maximum limit of 25 MB|
+| *10009* | The file is empty or doesn’t exist|
+| *10010* | The interaction contentLength field is empty|
+| *10011* | The number of interactions of type AttachmentInteraction does not match the number of attachment files|
+| *10012* | Enter a valid workItemId for the MessagingSession ID.|
```

Comment:
Since the interaction API is non-UI service, the word "Enter" here and several messages below are not proper, it should be either "Specify" or "Assign"